client/src/App.tsx

```diff
@@ -17,35 +17,47 @@ import RequestManagementContainer from "./components/RequestManagementContainer"
 export interface OwnProps {}
 
 interface StateProps {
-    a: number
-    user: User|null
+    a: number;
+    user: User|null;
 }
 
 type Props = StateProps & OwnProps;
 
 class App extends Component<Props, {}> {
-    async checkAuth(): Promise<User> {
-        return new Promise(resolve => setTimeout(() => resolve({
-            "uuid": "abcdedf-afdhkasdf-adfsk",
-            "name": "Evan Strat",
-            "isAdmin": true
-        }), 5000));
-    }
-
-    async componentWillMount(): Promise<void> {
-        const user: User = await this.checkAuth();
+    public async componentWillMount(): Promise<void> {
+        const userRequest = await fetch("/api", {
+            credentials: "include",
+            method: "POST",
+            headers: {
+                "Content-Type": "application/json",
+            },
+            body: JSON.stringify({
+                query: `
+                        query {
+                          user {
+                            uuid
+                            name
+                            admin
+                          }
+                        }
+                    `
+            }),
+        });
+        const json = await userRequest.json();
+        const user = json.data.user;
         console.log(user);
         if (user) {
             store.dispatch(setUser(user));
         }
     }
 
-    render() {
+
+    public render() {
         return (
             <div style={{
-                width: '100%',
+                width: "100%",
                 maxWidth: 960,
```

Comment:
I would put `px` to be explicit

Reply:
React assumes px if you don't put it, I remember getting a warning to remove px  (as it's unnecessary, or something) when I had it a while back.  I'll do some investigation though